consent-ws/src/main/resources/changesets/changelog-consent-36.0.xml

```diff
@@ -0,0 +1,15 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet author="grushton" id="114">
+        <addColumn tableName="consents">
+            <column name="dataUse" type="text"/>
+        </addColumn>
+        <sql dbms="hsql, mysql">UPDATE consents SET dataUse = ''</sql>
+        <sql dbms="hsql, mysql">ALTER TABLE consents MODIFY dataUse text NOT NULL</sql>
```

Comment:
mysql 5 doesn't let us create a `text` column with a default value in an `add column` statement, so manually doing the same thing with the extra sql statements.

Reply:
You could use `VARCHAR`, which does accept a default. I see you're using `TEXT` for all the other columns, so I think consistency is better and text is ok.

Reply:
Yes, I chose `text` to be consistent with how we're storing `useRestriction` currently.